src/toy/ERC20.sol

```diff
@@ -216,7 +216,7 @@ contract ERC20Base is Context, IERC20 {
    * - `to` cannot be the zero address.
    * - `from` must have a balance of at least `amount`.
    */
-  function _transfer(address from, address to, uint amount) internal virtual {
+  function _transfer(address from, address to, uint amount) internal virtual returns (bool) {
```

Comment:
if you change the interface you inherit from, this might not work with standard ERC20 contracts. Do openZep contracts return bool here?

Reply:
You are right, it does not look like the internal function for transfer returns a bool. https://docs.openzeppelin.com/contracts/2.x/api/token/erc20#ERC20-_transfer-address-address-uint256-